### ISSUE TITLE
i installed the beowser but the app thoast me no browser clients installed

### ISSUE THREAD

**Issue by 986244073** [ThirtyDegreesRay/OpenHub#372]
![Screenshot_20201116-112415](https://user-images.githubusercontent.com/16508677/99209650-9af73200-27fe-11eb-9891-6e8c22b2d718.jpg)
![Screenshot_20201116-112403](https://user-images.githubusercontent.com/16508677/99209651-9c285f00-27fe-11eb-8bd2-f24b41c1daee.jpg)

**Comment by platypusgit** [comment]
I get the same error. After clicking on the button to login, the yellow warning appears, that there is no browser detected. I have the SystemWebView and DuckDuckGo-Browser installed.

**Comment by Yquas** [comment]
@986244073 @platypusgit Maybe try installing more different browsers and look if it works with those.

**Comment by platypusgit** [comment]
@Yquas yeah, but that would just be a workaround to a bug and the issue is not gone. At least SystemWebView should work as the integrated system-standard in Android.

**Comment by Yquas** [comment]
@platypusgit Not really, I think. Because if the DuckDuckGo Browser doesn't support the action OpenHub wants to perform, it's not OpenHub's problem. And WebView won't help, because OpenHob never opens the OAuth screen inside itself. It just gives you the link. But maybe this could be added. The OAuth screen opening inside OpenHib so one doesn't need any other app except probably WebView.
